src/isomorphic-fetch.js

```diff
@@ -1,9 +1,16 @@
+/* globals require */
+
 import global from './metal/global';
 import isNodeLikeEnvironment from './metal/is-node-like-environment';
 
 const fetch = global.fetch;
 
 if (!fetch && isNodeLikeEnvironment()) {
-  global.fetch = global.require('node-fetch');
+  /* global.require is only available in the REPL 
+   * and Babel fails when you use require() explicitly
+   */
```

Comment:
hmm... i'm wondering, is it `amd` or `commonjs`? i thought the `commonjs` versions were loaded when the library is required/used in the nodejs environment.